### ISSUE TITLE
Temporal activity can have namespace attached to context

### ISSUE THREAD

**Issue by johncburns1** [nforgeio/neonKUBE#1020]
An activity context in Temporal can accept a namespace via the `workflow.WithWorkflowNamespace()` function.  It seems like this is used to execute an activity in a specific namespace with specific workers in that namespace.  Right now the `Neon.Temporal.ActivityRequest` does not pass a namespace as a part of the message.  I think that this might be a nice feature to have and pretty easy to implement.  For now I will just remove this line from the proxy.

**Comment by jefflill** [comment]
I believe we cover this on the .NET side with `ActivityOptions.Namespace`.  This is intended to override the client's default namespace when it's not null or empty.

**Comment by johncburns1** [comment]
Ok, I will leave this in then.  Maybe need to make sure to check this on the proxy side if it is not required.

**Comment by jefflill** [comment]
OK: I looked into this further.  The `ActivityOptions.Namespace` (same for Workflows) should be honored when executing an activity or workflow.  I expect that you're already doing this because this was standard Cadence behavior too (although I'm not sure we have a unit test for this case).

I was thinking for a moment that perhaps we should include the namespace in the workflow and activity messages but that doesn't make sense, because the worker specifies the namespace and all workflows/activities need to be relative to that.

**Comment by johncburns1** [comment]
I think that this activity option might be available for a case in which you would want to schedule an activity in a different namespace?

**Comment by jefflill** [comment]
Exactly.  We have the same thing for workflows.  There are three ways to specify the target namespace in the .NET client (in precedence order):

- namespace from the workflow or activity options 
- namespace from the  [WorkflowMethod] or [ActivityMethod] attribute decorating the target method
- namespace specified by client settings
